catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/api/EmailExtractor.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.monitor.api;
+
+import org.codice.ddf.catalog.ui.metacard.workspace.WorkspaceMetacardImpl;
+
+/**
+ * Get the email address for a workspace metacard.
+ */
+public interface EmailExtractor {
+
+    /**
+     * Get the email address for a workspace metacard.
+     *
+     * @param workspaceMetacard must be non-null
+     * @return email address (non-null)
+     */
+    String getEmail(WorkspaceMetacardImpl workspaceMetacard);
```

Comment:
Sorry, I should have removed it all together. I originally needed it to mock up the email address, but now emails are coming from the persistent store, not the metacard.